client/packages/system/src/Patient/CreatePatientModal/PatientFormTab.tsx

```diff
@@ -8,6 +8,7 @@ import {
 } from '@openmsupply-client/programs';
 import { PatientPanel } from './PatientPanel';
 import { createPatient, createPatientUI } from './DefaultCreatePatientJsonForm';
+import { isEqual } from 'lodash';
```

Comment:
as per #1271, lodash isn't tree shaking.
Also, as a general rule, we've been trying to not import from external packages in any package except common - in order to isolate / limit external packages & dependencies. It also means that we don't have duplicate imports in multiple packages - only into common, which then re-exports.

So.. could you add `isEqual` to `common/src/utils/index.ts` ( after the bundle size PR is merged! ) and import from 

`import { isEqual } from '@common/utils'`;

Reply:
Yup, looks like it was already there in `ObjUtils`, so have imported from there: b16aeb6d1

Reply:
I see the`useJsonForms` module imports the whole lodash `_` default. Should we fix that up too?